src/index.tsx

```diff
@@ -1,5 +1,10 @@
-import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
```suggestion
import { HashRouter } from 'react-router-dom';
```
Використовуй HashRouter, бо зараз при оновленні сторінки маємо помилку

https://user-images.githubusercontent.com/66570142/153867532-2c561a6c-6412-4841-9de3-3c1e5a5c066f.mov